insights/parsers/ls_etc.py

```diff
@@ -43,20 +45,22 @@
     []
     >>> ls_etc.total_of("/etc/sysconfig")
     96
+    >>> ls_etc.dir_entry('/etc/sysconfig', 'grub')
```

Comment:
One of the downsides of using `doctest` is that it compares literal string outputs, rather than semantic object representation.  Thus, the problem with testing the output of dictionaries is that the ordering of elements is arbitrary and can change.  It is better to test the type of the `dir_entry` return and check the existence of and look up items in the dictionary - and these are also more likely uses of the parser object in rules anyway.